democracy_club/templates/about.html

```diff
@@ -13,97 +13,103 @@ <h1>About</h1>
       {% filter markdown %}
 
 
-### Democracy Club aims to make the process of democracy better for everyone.
+## Democracy Club’s vision is of a society in which democracy thrives through knowledge, participation and openness.
 
-We believe that there is a lot of work needed to be done to bring the UK
-political system in the to digital age, and our tools are a start.
+### Our mission is to use open data, design and technology to give every citizen the information and participation opportunities they need, in a way that suits them.
 
-In this video from March 2016, Joe Mitchell — a co-ordinator at Democracy Club - explains how we try to do this.
+### We are non-partisan and we work openly. We’re a community of 10,000 volunteers and a small core team.
 
-<div class="u-responsive-video">
-	<iframe width="560" height="315" src="https://www.youtube.com/embed/efA1l9q7a1M" frameborder="0" allowfullscreen></iframe>
-</div>
+#### We don’t have a view of ‘the perfect democracy’. Instead, we try to make constant iterative improvements based on citizens’ needs. We judge those needs based on publicly available research, feedback on our websites and online search data.
 
-Some of the things we do are:
+# What we do
 
-* Organise a group of several thousand volunteers who want to make democracy
-  better. [Join them]({% url 'home' %})!
+## The club
 
-* Crowdsource and maintain a list of [candidates](https://candidates.democracyclub.org.uk/) for elections. [Contact us]({% url 'contact' %}) if
-  you are interested in using it.
+We coordinate a club of thousands of democracy volunteers, who do small online tasks to help make democracy better. The club help create our Data and Apps.
 
-* Run a [polling station finder](https://wheredoivote.co.uk/) to make it possible to find out where you can
-  vote, online. We work with local government to publish the data that powers
-  it.
+## Data
 
-* Build voter information services, like 2015’s YourNextMP and 2016's [WhoCanIVoteFor](https://whocanivotefor.co.uk). Together our services and data have helped over 1.3m people learn about their local candidates.
+We create the tools to aggregate data for democracy and make it available to everyone. This includes lists of elections, candidates and polling locations. Our data is trusted by the Electoral Commission, Google, Facebook, the Guardian and more.
 
-* Run a [register of elections](https://elections.democracyclub.org.uk/). Elections - especially by-elections - are happening regularly across the UK. But until now, nobody had developed an ID system that would allow us to capture them all and to run tools that work across many elections.
+## Apps
 
-* Collect [election leaflets](https://electionleaflets.org/) in a public archive. We ask the public to snap a
-  photo of leaflets put through their door so the world can see them.
+We build apps to answer direct questions from voters, such as 'Who can I vote for?' and 'Where do I vote?'.
```

Comment:
Should 'who...'/'where...' here be links to our sites/projects, or is that too link-heavy given we've linked to projects below?

Reply:
I prefer no links; keeps it cleaner.